hms-api/serverless.yml

```
@@ -43,6 +43,8 @@ provider:
     AZURE_AD_ISSUER: ${file(./secrets.json):AZURE_AD_ISSUER}
     AZURE_AD_CLIENT_PUBLIC_KEY: ${file(./public_key)}
     HACKATHON_TABLE: hackathon
+    HACKATHON_BY_TITLE_INDEX: hackathonById
```

I think you meant to do:
```suggestion
    HACKATHON_BY_TITLE_INDEX: hackathonByTitle
```

---

also, this by it's nature introduces a new requirement.  That hackathon Titles be globally unique, which means you need to validate this on creation and editing to ensure that you can still find a unique hackathon by title

---

same thing about the slugs, it now appears that they too must be unique 🤔

---

I'm basing this on how I imagine they are meant to be used, if I try to navigate to something like `www.hackathon.com/slug/hack4/` and I gave two things the slug `hack4` it won't know where to go

---

and now that I've looked more you don't seem to actually use the hackathon by title index anywhere

---

yeah I have to remove the title index again... wanted to check title and slug for uniqueness but half way through it jakob convinced me that slug would be enough and you could have 2 hackathons with the same name